o3d3xx/pcic/image_client.py

```diff
@@ -154,6 +156,12 @@ def readNextFrame(self):
 						diagnosticData['frameRate'] = struct.unpack('=I', bytes(data[20:24]))[0]
 					result['diagnostic'] = diagnosticData
 
+				elif chunkType == 401:
+					result['intrinsicCalibration'] = IntrinsicCalibration.from_buffer(data)
+
+				elif chunkType == 402:
+					result['inverseIntrinsicCalibration'] = IntrinsicCalibration.from_buffer(data)
+
```

Comment:
Since you have added the class for storing an extrinsic calibration as well: do you want to add a case here for parsing the corresponding chunk type 400, too? Then we have a complete package, calibration-wise.

Reply:
I added the chunk type 400 as well